internal/dataplane/kong_client.go

```diff
@@ -511,13 +512,39 @@ func (c *KongClient) maybeSendOutToKonnectClient(ctx context.Context, s *kongsta
 			c.logger.Error(err, "Skipped pushing configuration to Konnect")
 		} else {
 			c.logger.Error(err, "Failed pushing configuration to Konnect")
+			logKonnectErrors(c.logger, err)
 		}
 		return err
 	}
 
 	return nil
 }
 
+// logKonnectErrors logs details of each error response returned from Konnect API.
+func logKonnectErrors(logger logr.Logger, err error) {
+	if crudActionErrors := deckerrors.ExtractCRUDActionErrors(err); len(crudActionErrors) > 0 {
+		for _, actionErr := range crudActionErrors {
+			// TODO: log the CRUD operation (op type, entity kind and entity name)
+			// after we pass them in certain structures in GDR:
+			// https://github.com/Kong/go-database-reconciler/issues/41
```

Comment:
It's being done in this PR, isn't it? :)
```suggestion
```

Reply:
Done.